kong/plugins/file-log/schema.lua

```diff
@@ -10,7 +10,7 @@ return {
         fields = {
           { path = { description = "The file path of the output log file. The plugin creates the log file if it doesn't exist yet.", type = "string",
                      required = true,
-                     match = [[^[^*&%%\`]+$]],
+                     match = [[^[^%s*&%%\\`]+$]],
```

Comment:
This will introduce an issue where paths will not be allowed to contain spaces, not just at the beginning and end. For example, `/tmp/foo bar.log` would also become invalid. Is this the expected behavior?